linera-service/src/chain_listener.rs

```diff
@@ -25,6 +25,15 @@ pub struct ChainListenerConfig {
     pub(crate) delay_after_ms: u64,
 }
 
+impl ChainListenerConfig {
+    pub fn new(delay_before_ms: u64, delay_after_ms: u64) -> Self {
+        ChainListenerConfig {
+            delay_before_ms,
+            delay_after_ms,
+        }
+    }
+}
```

Comment:
If it's only used in the build script, I'd actually just make the fields `pub` instead of adding a constructor.